src/main/java/Duke.java

```diff
@@ -18,28 +18,45 @@ public static void main(String[] args) {
                        underscore ;
         System.out.println(intro);
 
-        // Creates an ArrayList
-        ArrayList<String> tasks = new ArrayList<String>();
+        // Creates an ArrayList of Task objects
+        ArrayList<Task> tasks = new ArrayList<Task>();
 
         // Creates scanner object to handle input
         Scanner in = new Scanner(System.in);
         String command = in.nextLine();
         String output = "";
+        int taskNum = -1;
 
         while ( !command.equals("bye") ){
 
             if ( command.equals("list")){
-                output = underscore;
+
+                output = underscore + "     Here are the tasks in your list:\n";
                 for (int i = 0; i < tasks.size(); i++){
-                    output +=  "     " + (i+1) + ". " + tasks.get(i) + "\n";
+                    output +=  "     " + (i+1) + ".[" + tasks.get(i).getStatusIcon() + "] " + tasks.get(i).description + "\n";
                 }
                 output += underscore;
 
                 System.out.println(output);
+
+            } else if ( command.substring(0,4).equals("done")){
```

Comment:
I think this might not be the best implementation. The bot can potentially mistake "donee" or "donekey" or "donevan" as command. A better implementation might be to split by the spacing using command.split(" "). We can then better identify the commands and the description without ambiguity.